accio-sqlrewrite/src/main/java/io/accio/sqlrewrite/AccioPlanner.java

```diff
@@ -24,15 +24,15 @@
 import java.util.List;
 
 import static io.accio.sqlrewrite.AccioSqlRewrite.ACCIO_SQL_REWRITE;
+import static io.accio.sqlrewrite.EnumRewrite.ENUM_REWRITE;
 import static io.accio.sqlrewrite.MetricViewSqlRewrite.METRIC_VIEW_SQL_REWRITE;
-import static io.accio.sqlrewrite.SyntacticSugarRewrite.SYNTACTIC_SUGAR_REWRITE;
 import static io.trino.sql.parser.ParsingOptions.DecimalLiteralTreatment.AS_DECIMAL;
 
 public class AccioPlanner
 {
     public static final List<AccioRule> ALL_RULES = List.of(
+            ENUM_REWRITE,
```

Comment:
Maybe we need to apply `ENUM_REWRITE` lastly because the enum maybe used in `column expression`.

Reply:
Nice catch, thx